translation_ab_experiment.py

```
@@ -0,0 +1,499 @@
+
+    return enriched
+
+
+def extract_translation_map(response_text: str, items: list[dict]) -> tuple[dict[str, str], str]:
+    if not response_text:
+        return {}, 'Missing response text.'
+    try:
+        batch_mod = _batch()
+        payload = batch_mod.parse_json_payload(response_text)
+        result_items = batch_mod.normalize_result_items(payload)
+    except Exception as exc:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)